### PR TITLE
Some Screens and GameAPI stuff

### PR DESCRIPTION
**Pull request by Mr-Auto**
Some leftovers from failed attempts at accessing the actual inputs in the game
Since i wrote down at least some of the variables types decide to make pr

### CODE REVIEW

**Comment by Mr-Auto** [comment]
If anyone want's to experiment, pattern for InputTable
```
.find_after_inst("41 0F 92 C1 48 C1 E0 05"_gh)
.decode_pc()
.at_exe()
```

**Comment by Dregu** [comment]
Nice we can use the input_index to assign a controller to a player spawned mid game from the Game->Players menu.

File: src/game_api/game_manager.hpp [hunk 1]
```diff
@@ -86,6 +86,25 @@ struct GameProps
     bool unknown9;
     bool unknown10;
     // there's more stuff here
+    std::array<size_t*, 12> unknown11; // pointers to something
```
Comment (Dregu) [inline src/game_api/game_manager.hpp:89]: And this is probably pointers to the 12 possible inputs (4kb, 4xinput, 4dinput?), something like maybe

```c++
struct Input {
  bool enabled;
  // padding
  uint8_t input_index;
  // padding
  INPUTS input;
}
```

**Comment by Dregu** [comment]
> If anyone want's to experiment, pattern for InputTable

Yeah my controllers seem to be setting stuff around there somewhere, and above it is a giant block of 00s that all the they keyboard keys are setting.

**Comment by Dregu** [comment]
I guess I'll merge this garbage first and add all the input stuff I figured out from your stuff to my pr...

**Pull request merged**